pkg/diagnostics/diagnostics.go

```diff
@@ -0,0 +1,56 @@
+package diagnostics
+
+import (
+	"encoding/json"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/keep-common/pkg/diagnostics"
+	"github.com/keep-network/keep-core/pkg/net"
+	"github.com/keep-network/keep-core/pkg/net/key"
+)
+
+var logger = log.Logger("keep-diagnostics")
+
+// Initialize set up the diagnostics registry and enables diagnostics server.
+func Initialize(
+	port int,
+) (*diagnostics.DiagnosticsRegistry, bool) {
+	if port == 0 {
+		return nil, false
+	}
+
+	registry := diagnostics.NewRegistry()
+
+	registry.EnableServer(port)
+
+	return registry, true
+}
+
+func RegisterConnectedPeersSource(registry *diagnostics.DiagnosticsRegistry, netProvider net.Provider) {
+
```

Comment:
Please remove this blank line and make sure all of the below lines doesn't exceed `80` columns of length [as per our go guidelines](https://github.com/keep-network/keep-core/blob/master/docs/development/go-guidelines.adoc#max-line-length)

Reply:
done